src/widgets/rating-menu/defaultTemplates.tsx

```diff
@@ -0,0 +1,86 @@
+/** @jsx h */
+import { h } from 'preact';
+
+import type { ComponentChild } from 'preact';
+import type {
+  RatingMenuComponentTemplates,
+  RatingMenuCSSClasses,
+} from './rating-menu';
+import { formatNumber } from '../../lib/formatNumber';
+import { cx } from '../../lib/utils';
+
+type ItemWrapperProps = { children: ComponentChild } & {
+  value: string;
+  count: number;
+  url: string;
+  cssClasses: RatingMenuCSSClasses;
+};
+
+function ItemWrapper({
+  children,
+  count,
+  value,
+  url,
+  cssClasses,
+}: ItemWrapperProps) {
+  if (count) {
+    return (
+      <a
+        className={cx(cssClasses.link)}
+        aria-label={`${value} & up`}
+        href={url}
+      >
+        {children}
+      </a>
+    );
+  }
+
+  return (
+    <div className={cx(cssClasses.link)} aria-label={`${value} & up`} disabled>
+      {children}
+    </div>
+  );
+}
+
+const defaultTemplates: RatingMenuComponentTemplates = {
+  item({ count, value, url, stars, cssClasses }) {
+    return (
+      <ItemWrapper
+        count={count}
+        value={value}
+        url={url}
+        cssClasses={cssClasses}
+      >
+        {stars.map((isFull, index) => (
+          <svg
+            key={index}
+            className={cx(
+              cssClasses.starIcon?.concat(
+                isFull ? 'cssClasses.fullStarIcon' : 'cssClasses.emptyStarIcon'
+              )
+            )}
```

Comment:
```suggestion
            className={cx(
              cssClasses.starIcon,
              isFull ? cssClasses.fullStarIcon : cssClasses.emptyStarIcon,
            )}
```

Reply:
Doesn't work, this `cx` is a temporary one to support the fact that `cssClasses` can be `string[]` (which we should drop in v5). This `cx` doesn't take variadic arguments, only a single one that's a `string` or a `string[]`.

Reply:
Ah I see! In this case if we're sure `cssClasses.starIcon` is an array that's fine. I updated the suggestion to return the values of `cssClasses.fullStarIcon` and `cssClasses.emptyStarIcon` in the ternary.

```suggestion
            className={cx(
              cssClasses.starIcon?.concat(
                isFull ? cssClasses.fullStarIcon : cssClasses.emptyStarIcon
              )
            )}
```

Reply:
Damn, I missed this 🤦‍♀️ Fixed in [ac6464f](https://github.com/algolia/instantsearch.js/pull/5092/commits/ac6464ff631592a03a96d9e16fab7b8aa6091961).